braid/braid.h

```diff
@@ -731,6 +741,15 @@ braid_SetSpatialRefine(braid_Core         core,   /**< braid_Core (_braid_Core)
                        braid_PtFcnSRefine srefine /**< function pointer to spatial refinement routine */
                        );
 
```

Comment:
This comment before "SetSync" is what get's automatically put into the Users manual.  So, we should make sure that it's good.  Maybe something like,

* Set sync routine with user-defined routine.
* The sync routine gives the user access to XBraid and to the user's app 
* at various points (primarily once per iteration inside FRefine and outside
*  in the main cycle loop).  This function is called once per-processor (not for every 
* state vector stored on that processor, like access).  The envisioned usage here
* is to allow the user to update their app once per iteration based on information
* from XBraid.  This has been useful for time-space adaptivity. 
* Default is no sync routine

Reply:
Changed in https://github.com/XBraid/xbraid/pull/27/commits/8fc6b99c17f8feb553d0b48a5ee5c9b8c567b981